durabletask/internal/shared.py

```diff
@@ -18,13 +20,15 @@ def get_default_host_address() -> str:
     return "localhost:4001"
 
 
-def get_grpc_channel(host_address: Union[str, None]) -> grpc.Channel:
+def get_grpc_channel(host_address: Union[str, None], metadata: List[Tuple[str, str]]) -> grpc.Channel:
     if host_address is None:
         host_address = get_default_host_address()
     channel = grpc.insecure_channel(host_address)
+    if metadata != None and len(metadata) > 0:
```

Comment:
nit: I don't think we use `!= None` much in this codebase.
```suggestion
    if metadata is not None and len(metadata) > 0:
```

Reply:
addressed